packages/kit/src/exports/internal/event-state.js

```diff
@@ -10,23 +10,35 @@ export const EVENT_STATE = Symbol('remote');
  * 	prerendering: PrerenderOptions | undefined
  *  transport: ServerHooks['transport'];
  *  handleValidationError: ServerHooks['handleValidationError'];
+ *  tracing: {
+ *    record_span: RecordSpan
+ *  }
  *  form_instances?: Map<any, any>;
  * 	remote_data?: Record<string, MaybePromise<any>>;
  *  refreshes?: Record<string, any>;
  * }} RequestEventState
  */
 
 /**
- * @param {SSRState} state
- * @param {SSROptions} options
- * @returns {RequestEventState}
+ * @param {{ event: RequestEvent, state: SSRState, options: SSROptions, record_span: RecordSpan }} args
+ * @returns {RequestEvent}
  */
-export function create_event_state(state, options) {
-	return {
-		prerendering: state.prerendering,
-		transport: options.hooks.transport,
-		handleValidationError: options.hooks.handleValidationError
-	};
+export function add_event_state({ event, state, options, record_span }) {
+	Object.defineProperty(event, EVENT_STATE, {
+		value: {
+			prerendering: state.prerendering,
+			transport: options.hooks.transport,
+			handleValidationError: options.hooks.handleValidationError,
+			// this is necessary to avoid importing `record_span` in `sequence`, which
+			// won't work because `record_span` depends on `otel`, which depends on
+			// being bundled through Vite.
+			tracing: {
+				record_span
+			}
+		},
+		enumerable: false
```

Comment:
`false` is the default, it can be omitted

```suggestion
		}
```